src/drafts/SelectPanel2/SelectPanel.tsx

```diff
@@ -52,6 +52,10 @@ const SelectPanel = props => {
   // sync open state
   React.useEffect(() => setInternalOpen(props.open), [props.open])
 
+  const onInternalOpen = () => {
+    if (props.open === undefined) setInternalOpen(true)
```

Comment:
Undo change of current behavior

```suggestion
    setInternalOpen(true)
```